lib/customisation.nix

```
@@ -55,6 +59,11 @@ rec {
      injects `override` attribute which can be used to override arguments of
      the function.
 
+     Please refer to "Nixpkgs Contributors Guide" section
+     "<pkg>.overrideDerivation" to learn about `overrideDerivation` and caveats
```

```suggestion
     Please refer to  documentation on [`<pkg>.overrideDerivation`](#sec-pkg-overrideDerivation) to learn about `overrideDerivation` and caveats
```

This should render cleanly in the HTML manual.